frontend/src/components/helpers/AccessManager/index.tsx

```diff
@@ -51,7 +51,15 @@ const AccessManager: FC<IAccessManagerProps> = (
 
     const checkEndpoint = () => {
         const endpoints = permissions.endpoints;
-        if (endpoint && endpoints && endpoints.includes(endpoint)) {
+        if (Array.isArray(endpoint)) {
+            for (const endp of endpoint) {
+                if (endpoints.includes(endp)) {
```

Comment:
It would be always true?

Reply:
? No, only if passed endpoint is array and at least one of them is permited. Like path                                       
["/response/:id/", "/response/:id/modify/:responseId/"]